src/core/models.py

```diff
@@ -245,8 +262,8 @@ def sans(self) -> dict[str, list[str]]:
             return {}
 
         return {
-            "sans_ip": [self.private_ip],
-            "sans_dns": [self.hostname, self.fqdn],
+            "sans_ip": [ip for ip in [self.private_ip] if ip],
```

Comment:
```suggestion
            "sans_ip": self.private_ip,
```

Reply:
Again, this is due to the lint

Reply:
Returned to `"sans_ip": [self.private_ip],`